SteamKit2/SteamKit2/Base/PacketBase.cs

```diff
@@ -95,15 +102,28 @@ public sealed class PacketClientMsgProtobuf : IPacketMsg
         /// <value>
         /// The target job id.
         /// </value>
-        public ulong TargetJobID { get; }
+        public ulong TargetJobID => Header.Proto.jobid_target;
         /// <summary>
         /// Gets the source job id for this packet message.
         /// </summary>
         /// <value>
         /// The source job id.
         /// </value>
-        public ulong SourceJobID { get; }
-
+        public ulong SourceJobID => Header.Proto.jobid_source;
+        /// <summary>
+        /// Gets the header for this packet message.
+        /// </summary>
+        /// <value>
+        /// The header.
+        /// </value>
+        public MsgHdrProtoBuf Header { get; }

```

Comment:
I'm wondering if this should be `internal`. Previously immutable properties are now mutable through this.